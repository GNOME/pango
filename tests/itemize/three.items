<span font="Cantarell 11" fallback="false">عַرַبִ</span>
