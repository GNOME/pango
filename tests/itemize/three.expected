<span font="Cantarell 11" fallback="false">عַرַبִ</span>

Items:  عַرַبִ       
Chars:  6(0)               
Font:   Cantarell 11       
Script: arabic             
Lang:   ar                 
Bidi:   1                  
Attrs:  0 12 fallback false
